<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>farmaceutico_agregar_medicamento</class>
 <widget class="QDialog" name="farmaceutico_agregar_medicamento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nombre_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
border-bottom: 1px solid gray;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tipo_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
border-bottom: 1px solid gray;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="costo_lineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
border-bottom: 1px solid gray;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Costo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Premedicado:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="descripcion_plainTextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid gray;
border-radius: 5px;</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Descripci√≥n:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="cantidad_spinBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QCheckBox" name="si_checkBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>Si</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="no_checkBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>85</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: #EF1F4B;
	color: white;
	border: 1px solid #EF1F4B;
	border-radius: 5px;
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agregar_medicamento">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
